pipelines/concourse-rclone-docker-image/docker-image-pipeline.yml

```diff
@@ -0,0 +1,26 @@
+---
+resources:
+  - name: concourse-rclone-resource
+    type: git
+    source:
+      uri: https://github.com/mitodl/concourse-rclone-resource.git
+      branch: mitodl
```

Comment:
Why not stick with main or master?

Reply:
Just because I thought it would be easier to make a PR to change the upstream this way, or to handle incoming pulls from upstream without our commits being on `master`. But maybe that's not such a big deal.

Reply:
Not a big deal either way, let's merge as is so as not to block on this. We can always change things later.